x-pack/test/functional_with_es_ssl/apps/cases/list_view.ts

```diff
@@ -240,6 +240,33 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
           await cases.casesTable.validateCasesTableHasNthRows(1);
           await testSubjects.exists('case-user-profile-avatar-cases_all_user2');
         });
+
+        it('filters cases without assignees', async () => {
+          await cases.casesTable.openAssigneesPopover();
+          await cases.common.selectFirstRowInAssigneesPopover();
+          await cases.casesTable.validateCasesTableHasNthRows(2);
+
+          const firstCaseTitle = await (
+            await cases.casesTable.getCaseFromTable(0)
+          ).findByTestSubject('case-details-link');
+
+          const secondCaseTitle = await (
+            await cases.casesTable.getCaseFromTable(1)
+          ).findByTestSubject('case-details-link');
+
+          expect(await firstCaseTitle.getVisibleText()).be('test2');
+          expect(await secondCaseTitle.getVisibleText()).be('matchme');
+        });
+
+        it('filters cases without assignees', async () => {
```

Comment:
This test name is the same as above. Is it testing that one of the cases is returned because it matches an assignee? Maybe update it to `filters cases with and without assignees`.